rotkehlchen/history/events/structures/base.py

```diff
@@ -264,18 +264,18 @@ def get_type_identifier(self, **kwargs: Any) -> int:  # pylint: disable=unused-a
     def serialize(self) -> dict[str, Any]:
         """Serialize the event alone for api"""
         serialized_data = {
+            'event_type': self.event_type.serialize(),
```

Comment:
Can you move it to the following order? 

1. timestamp
2. type
3. subtype
4. location
5. asset_id
6. symbol/name
7. amount
8. fiat_value
9. notes
10. identifier
11. entry_type
12. event_identifier
13. sequence_index

The reason for the following order is to put time first. Since many spreadsheets are ordered by it. Then in order the important columns to identify the event by. That's until The notes (9).

Then just put the other stuff they may need but probably don't care about.